### PR TITLE
Fix/2121/aligment badge counter

### LINKED ISSUE #2121: Badge counter wrong alignment
**Describe the bug**
Badge counter have a wrong alignment when it have 1 or 2 characters or non content.
From SUi demo:
It seems like it have the correct position when is full "+99"
image

**Steps to reproduce**
https://sui-components.vercel.app/workbench/molecule/badgeCounter/demo

**Expected behaviour**
First position should be the same like when is full now.
And when is full it have to increase the size to the right (not to the left like now)
image

### PR DESCRIPTION
**Pull request by IsabelGomez87**
#### `🔍 Show`

**TASK**: #2121 

### Types of changes

- [x] 🪲 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [x] 💄 Styles

### Description, Motivation and Context
Fix the badge counter alignment to the respect children

### Screenshots - Animations
image

### CODE REVIEW

**Review by nucliweb — APPROVED** [review]

File: components/molecule/badgeCounter/src/styles/settings.scss [hunk 1]
```diff
@@ -13,20 +13,20 @@ $z-molecule-badge-counter: 1 !default;
...
 $molecule-badge-counter-small-offset-top: -4px !default;
-$molecule-badge-counter-small-offset-right: -7px !default;
+$molecule-badge-counter-small-offset-right: -6px !default;
 
 /* medium */
 $molecule-badge-counter-medium-diameter: 16px !default;
 $molecule-badge-counter-medium-diameter-dot: 5px !default;
 $molecule-badge-counter-medium-offset-top: -6px !default;
-$molecule-badge-counter-medium-offset-right: -12px !default;
+$molecule-badge-counter-medium-offset-right: -8px !default;
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/styles/settings.scss]:
```suggestion
$molecule-badge-counter-medium-offset-right: -$sz-base !default;
```

File: components/molecule/badgeCounter/src/styles/settings.scss [hunk 2]
```diff
@@ -13,20 +13,20 @@ $z-molecule-badge-counter: 1 !default;
 $molecule-badge-counter-small-diameter: 12px !default;
 $molecule-badge-counter-small-diameter-dot: 4px !default;
 $molecule-badge-counter-small-offset-top: -4px !default;
-$molecule-badge-counter-small-offset-right: -7px !default;
+$molecule-badge-counter-small-offset-right: -6px !default;
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/styles/settings.scss]:
```suggestion
$molecule-badge-counter-small-offset-right: -($sz-base - $sz-base * 0.25) !default;
```

File: components/molecule/badgeCounter/src/styles/settings.scss [hunk 3]
```diff
@@ -13,20 +13,20 @@ $z-molecule-badge-counter: 1 !default;
...
-$molecule-badge-counter-medium-offset-right: -12px !default;
+$molecule-badge-counter-medium-offset-right: -8px !default;
 $p-molecule-badge-counter-medium: 0 $p-s !default;
 
 /* large */
 $molecule-badge-counter-large-diameter: 24px !default;
 $molecule-badge-counter-large-diameter-dot: 6px !default;
 $molecule-badge-counter-large-offset-top: -12px !default;
-$molecule-badge-counter-large-offset-right: -22px !default;
+$molecule-badge-counter-large-offset-right: -12px !default;
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/styles/settings.scss]:
```suggestion
$molecule-badge-counter-large-offset-right: -($sz-base + $sz-base * 0.5) !default;
```

**Review by andresin87 — CHANGES_REQUESTED** [review]

File: components/molecule/badgeCounter/src/styles/settings.scss (hunk 3 above)
Comment (IsabelGomez87) [reply components/molecule/badgeCounter/src/styles/settings.scss]: done!

File: components/molecule/badgeCounter/src/styles/settings.scss (hunk 2 above)
Comment (IsabelGomez87) [reply components/molecule/badgeCounter/src/styles/settings.scss]: done!

File: components/molecule/badgeCounter/src/styles/settings.scss (hunk 1 above)
Comment (IsabelGomez87) [reply components/molecule/badgeCounter/src/styles/settings.scss]: done!

**Review by andresin87 — APPROVED** [review]

**Pull request merged**